x-pack/plugins/event_log/server/es/init.ts

```diff
@@ -35,6 +37,95 @@ class EsInitializationSteps {
     this.esContext = esContext;
   }
 
+  async setExistingIndexTemplatesToHidden() {
+    try {
```

Comment:
The way these `set*()` methods are written, they will stop when they hit the first error.  Seems like we should just be processing each one, and just logging if we have an error on a particular resource.  So putting a separate try/catch around each `this.esContext.esAdapter.someMethod()` call.  Or we could change the methods to not throw an error, and just log.

Reply:
Done in this commit: https://github.com/elastic/kibana/pull/110929/commits/b2ca92d664bbf022e0ac9b3815e85580572445f5